python/ray/rllib/agents/agent.py

```diff
@@ -374,10 +377,17 @@ def _setup(self, config):
 
     @override(Trainable)
     def _stop(self):
+        # Call stop on all evaluators to release resources
+        if hasattr(self, "local_evaluator"):
+            self.local_evaluator.stop()
+        if hasattr(self, "remote_evaluators"):
+            ray.get([ev.stop.remote() for ev in self.remote_evaluators])
```

Comment:
note: don't necessarily need the ray.get() here, which could take a while to execute in a blocking way when there are many workers. It should be fine to let it execute async (similar to __ray_terminate__ below).

Reply:
Done